gui/src/theme/mod.rs

```diff
@@ -11,6 +11,7 @@ use iced::{
     },
 };
 
+#[allow(unused)]
```

Comment:
If you don't use it, why would you keep it?

Reply:
to avoid reimplement it in case we want use other kind items, but i can remove all unused  id you prefer

Reply:
Yeah, we should not keep dead code laying around.

Reply:
done